src/nft/components/bag/BagRow.css.ts

```diff
@@ -153,23 +156,25 @@ export const bagRowPrice = style([
 
 export const assetName = style([
   sprinkles({
-    fontSize: '14',
-    fontWeight: 'semibold',
+    fontSize: '16',
+    fontWeight: 'medium',
     overflow: 'hidden',
     textOverflow: 'ellipsis',
     whiteSpace: 'nowrap',
   }),
   {
-    lineHeight: '20px',
+    lineHeight: '24px',
```

Comment:
Left a comment on the Figma but standard Subhead typography should have a lineheight of 16 as per Uni standards